docs/swagger.json

```
@@ -20,7 +20,69 @@
                 "summary": "Returns details of the configuration of eBPF Programs for all interfaces on a node",
                 "responses": {
                     "200": {
-                        "description": ""
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/l3af/configs/v1/add": {
+            "post": {
+                "description": "Adding new eBPF Programs on node",
```

```suggestion
                "description": "Adds new eBPF Programs on node",
```
use grammar consistent with line 13